runtime/queries/beancount/highlights.scm

```diff
@@ -0,0 +1,35 @@
+(date) @variable.builtin
+(txn) @variable.builtin
+
+(account) @type
+
+[
+  (amount)
+  (incomplete_amount)
+  (amount_tolerance)
+  (number)
+] @constant.numeric
+
+
+(key) @label
+(string) @string
+
+[
+  (currency)
+  (tag)
+  (link)
+] @constant
+
+(comment) @comment
+
+[
+  (minus)
+  (plus)
+] @keyword.operator
```

Comment:
I think this makes more sense as `@operator` but I'm not too familiar with beancount

Reply:
It's been a while since I've used beancount myself, along with being new to tree-sitter, so I'm unsure as well. :/

Reply:
I want my minus sign to show the difference from the following number 

like 
![a1](https://user-images.githubusercontent.com/716514/183003029-e3fc13e4-801a-45b2-98bb-de249ee606a4.jpg)

Reply:
@erasin Does that mean you want it to be `@operator`? I'll defer to you on this one.

Reply:
If this is a spacing issue, then `bean-format` looks like it does a good job.

```toml
[[language]]
name = "beancount"

[language.formatter]
command = "bean-format"
```

I haven't gotten the LSP to work, but we'll see.

Reply:
I also support that mark minus with `@operator` .

Reply:
Updated and pushed.